lib/cmd/fh3/credentials.js

```diff
@@ -22,20 +22,16 @@ function credentials(argv, cb) {
   var action = args[0];
   if ("list" === action) {
     return listCredentials(args, cb);
-  } /*else if ("create" === action) {
-    return createCredentials(args, cb);
-  } else if ("update" === action) {
-    return updateCredentials(args, cb);
-  } else if ("delete" === action) {
-    return deleteCredentials(args, cb);
-  }*/ else {
+  } else {
```

Comment:
The module doesn't have the other CRUD functions available, maybe this comment was left like a TODO or a template?

I'm not familiar with the back-end yet, so I don't know if the other endpoints are there.